starknet_py/contract.py

```diff
@@ -515,10 +515,10 @@ async def from_address(
             abi = await ContractAbiResolver(
                 address=address, client=client, proxy_config=proxy_config
             ).resolve()
-        except UnsupportedAbiError:
+        except UnsupportedAbiError as err:
             raise ValueError(
                 "Provided address of Cairo1 contract which is currently not supported in Contract."
-            )
+            ) from err
```

Comment:
This seems like a duplicate of your another PR. Are you sure the target branch is correct?

Reply:
This specific change has been in another PR, because the lint problem is present on the current target branch (`update/cairo-0.11`).

Reply:
Just make sure it doesn't break anything when merging (it shouldn't I think).